textdb/textdb-textql/src/test/java/edu/uci/ics/textdb/textql/languageparser/TextQLParserTest.java

```diff
@@ -216,11 +216,11 @@ public void testIdentifierListToListString() throws ParseException {
         Assert.assertEquals((new TextQLParser(string2InputStream(" B7FROM "))).identifierListToListString(), Arrays.asList("B7FROM"));
         Assert.assertEquals((new TextQLParser(string2InputStream(" A "))).identifierListToListString(), Arrays.asList("A"));
         Assert.assertEquals((new TextQLParser(string2InputStream(" FROMa "))).identifierListToListString(), Arrays.asList("FROMa"));
-        assertException(()->(new TextQLParser(string2InputStream(" j7i,/8s7s/ "))).identifierListToListString(), ParseException.class);
-        assertException(()->(new TextQLParser(string2InputStream(" k8i,\"8s7s\" "))).identifierListToListString(), ParseException.class);
-        assertException(()->(new TextQLParser(string2InputStream(" k8i,,k9j "))).identifierListToListString(), ParseException.class);
-        assertException(()->(new TextQLParser(string2InputStream(" k8i,/8s7s/ "))).identifierListToListString(),ParseException.class);
-        assertException(()->(new TextQLParser(string2InputStream(" k8i, "))).identifierListToListString(), ParseException.class);
+        Assert.assertEquals((new TextQLParser(string2InputStream(" j7i,/8s7s/ "))).identifierListToListString(), Arrays.asList("j7i"));
+        Assert.assertEquals((new TextQLParser(string2InputStream(" k8i,\"8s7s\" "))).identifierListToListString(), Arrays.asList("k8i"));
+        Assert.assertEquals((new TextQLParser(string2InputStream(" k8i,,k9j "))).identifierListToListString(), Arrays.asList("k8i"));
+        Assert.assertEquals((new TextQLParser(string2InputStream(" k8i,/8s7s/ "))).identifierListToListString(), Arrays.asList("k8i"));
+        Assert.assertEquals((new TextQLParser(string2InputStream(" k8i, "))).identifierListToListString(), Arrays.asList("k8i"));
```

Comment:
By having a bigger number of look-ahead tokens, some previous parse errors can be avoided. These changes reflect that.